modules/install-osp-deploy-dualstack.adoc

```diff
@@ -164,4 +164,11 @@ ipv6.addr-gen-mode=0
 ----
 
 After you create and edit the file, reboot the installation host.
+====
+
+[NOTE]
+====
+The `ip=dhcp,dhcp6` kernel argument, which is set to all the nodes, generates a Network Manager connection profile that is activated on multiple interfaces simultaneously.
```

Comment:
"set to" or "set for"?

Reply:
I think it's set to

Reply:
Please add "results in a single NetworkManager connection profile" because "single" is the main culprit here and why this doc page is written